samples/nrf9160/agps/prj_pgps.conf

```diff
@@ -0,0 +1,86 @@
+#
```

Comment:
It'd actually make sense to make this an actual overlay file, since it's only adding extra configs on top of the default `prj.conf`. Only additional configs need to be present there, no need to repat the ones from `prj.conf`. And a typical convention is to use `overlay-*.conf` name pattern for such files, I suggest `overlay-pgps.conf`.

Reply:
Great, good advice.  Thanks.

Reply:
I decided to create an additional overlay, for agps+pgps.  In order to make the overlay-pgps.conf vs. overlay-agps-pgps.conf understandable, I want to include the AGPS-related configs even if they are redundant with the ones in the base prj.conf.